beacon-chain/blockchain/service.go

```diff
@@ -130,11 +128,11 @@ func (c *ChainService) IncomingBlockFeed() *event.Feed {
 // persisted to the db.
 func (c *ChainService) HasStoredState() (bool, error) {
 
-	hasActive, err := c.beaconDB.Has([]byte(activeStateLookupKey))
+	hasActive, err := c.beaconDB.Has([]byte(ActiveStateLookupKey))
```

Comment:
I don't think this is relevant since we are not storing active state in db

Reply:
we compute the active state whenever we receive a block right ? in that case I will remove all references to persisting active state in the repo